app/assets/stylesheets/application.scss

```diff
@@ -37,6 +36,5 @@
 * =require chosen
 * =require select2
 * =require select2-custom
```

Comment:
@dougmartin shouldn't this select2 and select2-custom be removed as well? Or is something else using them?

Reply:
I thought that select2-custom was being used by react-select but I see that it isn't now so I'll remove  both of the (along with the select2 gem).